alerts/template_test.go

```diff
@@ -93,3 +153,115 @@ func TestFormatTitleSimple(t *testing.T) {
 		})
 	}
 }
+
+func TestFormatLabelsSimple(t *testing.T) {
+	msg := webhook.Message{
+		Data: &amtmpl.Data{
+			Status: "firing",
+			Alerts: []amtmpl.Alert{
+				{
+					Annotations: amtmpl.KV{"env": "prod", "svc": "foo"},
+					Labels: amtmpl.KV{"cluster": "rick", "namespace": "opf-ci-pipelines"},
+				},
+				{
+					Annotations: amtmpl.KV{"env": "stage", "svc": "foo"},
+					Labels: amtmpl.KV{"cluster": "smaug", "namespace": "openshift-monitoring"},
+				},
+			},
+		},
+	}
+	var expectOutputTest1 = make([]string, 0)
+	var expectOutputTest2 = make([]string, 1)
+	expectOutputTest2[0] = "firing"
+	var expectOutputTest3 = make([]string, 1)
+	expectOutputTest3[0] = "firing"
+	var expectOutputTest4 = make([]string, 4)
+	expectOutputTest4[0] = "rick"
+	expectOutputTest4[1] = "opf-ci-pipelines"
+	expectOutputTest4[2] = "smaug"
+	expectOutputTest4[3] = "openshift-monitoring"
+	var expectOutputTest5 = make([]string, 2)
+	expectOutputTest5[0] = "prod"
+	expectOutputTest5[1] = "stage"
+	var expectOutputTest6 = make([]string, 0)
```

Comment:
I would get rid of these assignments and include them directly in the struct creations, e.g.: 

 ```
 {
	testName: "test status directly", 
	labelsTmpl: "{{.Status}}", 
	expectErrTxt: "", 
	expectOutput: []string{"firing"},   // get rid of expectOutputTest3 entirely
}
 ```

Reply:
also once again make these outputs generic, prod/stage is fine, but `openshift-monitoring/rick/opf-ci-pipelines/etc` are operate-first specific.